conda_forge_tick/migrators.xsh

```diff
@@ -675,7 +715,7 @@ class Rebuild(Migrator):
 
 
 class CompilerRebuild(Rebuild):
-    bump_number = 1000
+    bump_number = 1001
```

Comment:
I guess this will cause everything to get a rebuild on main and gcc7?

Reply:
Didn't mean to commit that